<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsApplicationModuleEditorClass</class>
 <widget class="QDialog" name="CwmsApplicationModuleEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module bearbeiten</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Modul</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="4">
       <widget class="QPushButton" name="m_pqleSelectData">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/edit24.png</normaloff>:/icons/edit24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pqleData">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Daten</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pqleForm">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="m_pqpbSelectForm">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/edit24.png</normaloff>:/icons/edit24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Formular</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_pqrbObjectList">
        <property name="text">
         <string>Objektcontainer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pqleView">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="m_pqrbObject">
        <property name="text">
         <string>Objekt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_pqlView">
        <property name="text">
         <string>Sicht</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="m_pqpbSelectView">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/edit24.png</normaloff>:/icons/edit24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="m_pqpbDeleteView">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/DeleteDocument.png</normaloff>:/icons/DeleteDocument.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="m_pqpbDeleteForm">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/DeleteDocument.png</normaloff>:/icons/DeleteDocument.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="m_pqpbDeleteData">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../WMSadmin/resources/emresources.qrc">
          <normaloff>:/icons/DeleteDocument.png</normaloff>:/icons/DeleteDocument.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOk">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="m_pqpbSelectIcon">
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Modul bearbeiten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="m_pqchbActive">
     <property name="text">
      <string>Aktiv</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pqleName"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_pqchbDefault">
     <property name="text">
      <string>Standard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOk</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqleSelectData</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>SelectDataClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbSelectForm</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>SelectFormClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbSelectView</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>SelectViewClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbSelectIcon</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>SelectIconClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbDeleteView</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>DeleteViewClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbDeleteForm</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>DeleteFormClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbDeleteData</sender>
   <signal>clicked()</signal>
   <receiver>CwmsApplicationModuleEditorClass</receiver>
   <slot>DeleteDataClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OKClickedSlot()</slot>
  <slot>SelectDataClickedSlot()</slot>
  <slot>SelectFormClickedSlot()</slot>
  <slot>SelectViewClickedSlot()</slot>
  <slot>SelectIconClickedSlot()</slot>
  <slot>DeleteViewClickedSlot()</slot>
  <slot>DeleteFormClickedSlot()</slot>
  <slot>DeleteDataClickedSlot()</slot>
 </slots>
</ui>
